entrypoint.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+ETC_HOSTS_PROXY_BIND_ADDRESS="${ETC_HOSTS_PROXY_BIND_ADDRESS:-0.0.0.0:8080}"
+ETC_HOSTS_PROXY_MODE="${ETC_HOSTS_PROXY_MODE:-http}"
+ADDITIONAL_ARGS=""
+
+if [ ! -z "ETC_HOSTS_PROXY_BIND_ADDRESS" ]; then
```

Comment:
Let me quote the `shellcheck` tool here :)
> SC2157 (error): Argument to -z is always false due to literal strings.

Reply:
Also
> SC2236 (style): Use -n instead of ! -z.

Reply:
or maybe you had `-v` (which checks the variable is defined) in mind instead of `-z` (which checks if an expanded string is empty)..

Reply:
yeah, sorry write on rush and without checks %) fixed